### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by benlansdell** [benlansdell/ethome#3]
From imcatta JOSS review: 

Documentation

    I struggled a little in following the code presented in the README.
        dataset.features.add('cnn1d_prob') If I run this on the nwb example dataset, as suggested above, it tells me that DataFrame doesn't have necessary columns to compute this set of features.. Also, I can’t find which columns are needed. Please consider stating in the documentation what columns are needed and how to get those in the sample dataset.
        cross_val_score(..., dataset.ml.labels, ...). No labels are present in the nwb example dataset, so this is not working with the dataset proposed above.
        Please add package requirements (e.g. Python versions supported) to the README.
        “For this, you must have labeled your body parts in a certain way (refer to How To)”. Please add the link to the How to section.
        Please add a link to the web documentation in the README.
    It is stated that “dataset.features.add('cnn1d_prob')Uses a pretrained CNN to output probabilities of 3 behaviors (attack, mount, social investigation).”. More information on this is needed. How was this CNN built? Also, I found out that the repository contains 5 pre-trained models, but no information on that is provided.
    Some users may not be fully confident with the DLC, SLEAP, NWB, BORIS and NWB file formats. Please explain the different data input formats supported in more detail.
    Guidelines for report issues or problems with the software and seek support are missing.
    In How To guide, section “3a In-built support for resident-intruder setup”, more information on what is MARS and context should be provided.

**Comment by benlansdell** [comment]
I have made the following changes in response to these comments on documentation:

* dataset.features.add('cnn1d_prob') If I run this on the nwb example dataset, as suggested above, it tells me that DataFrame doesn't have necessary columns to compute this set of features.. Also, I can’t find which columns are needed. Please consider stating in the documentation what columns are needed and how to get those in the sample dataset.
    * This error now prints which columns are required. Also, that section in the readme now gets its data from the DLC/BORIS dataset, which does have the necessary columns. 

* cross_val_score(..., dataset.ml.labels, ...). No labels are present in the nwb example dataset, so this is not working with the dataset proposed above.
    * The DLC/BORIS dataset, which does have labels, has replaced the nwb one, for this part of the quickstart guide. Hopefully this makes this section clearly to run through

* Please add package requirements (e.g. Python versions supported) to the README.
    * Added python versions this has been tested on

* “For this, you must have labeled your body parts in a certain way (refer to How To)”. Please add the link to the How to section.
    * DONE

* Please add a link to the web documentation in the README.
    * DONE
    
* It is stated that “dataset.features.add('cnn1d_prob')Uses a pretrained CNN to output probabilities of 3 behaviors (attack, mount, social investigation).”. More information on this is needed. How was this CNN built? Also, I found out that the repository contains 5 pre-trained models, but no information on that is provided.
    * In the How To guide, a note has been added: "`cnn1d_prob` runs a 1D CNN and outputs prediction probabilities of three behaviors (attack, mount, and investigation). Even if you're not interested in these exact behaviors, they may still be useful for predicting the occurrence of other behaviors, as part of an ensemble model. The details of this pretrained model are found here: [https://github.com/benlansdell/mabetask1_ml](https://github.com/benlansdell/mabetask1_ml). " This, in addition with the context provided about the MARS dataset and support for resident-intruder, should make things clearer.
    
* Some users may not be fully confident with the DLC, SLEAP, NWB, BORIS and NWB file formats. Please explain the different data input formats supported in more detail.
    * I have added a section to the readme that does introduce each of these formats briefly, and provides links to where more info can be found.
    
* Guidelines for report issues or problems with the software and seek support are missing.
    * In README, added a note pointing to CONTRIBUTING.md file which describes this
    
* In How To guide, section “3a In-built support for resident-intruder setup”, more information on what is MARS and context should be provided.
    * This has been added: Paragraph now reads: "First, if your setup is a social mouse study, involving two mice, similar enough to the standard resident-intruder setup, then you can use some pre-designed feature sets. The body parts that are tracked must be those from the MARS dataset [1] (See Figure). The MARS dataset is the Mouse Action Recognition System dataset, published by Segalin et al. It is a large, annotated mouse pose dataset that is useful for building ML models of mouse behavior. `ethome` uses pretrained models and feature sets that have been demonstrated to perform well at mouse action recognition on this dataset -- a combination of these models and features placed 3rd in AICrowd's [Multiagent behavior contest](https://www.aicrowd.com/challenges/multi-agent-behavior-representation-modeling-measurement-and-applications) in 2021."
